comment.py

```diff
@@ -14,7 +14,7 @@
     agent = CommentAgent(
         repo_name=REPO_NAME,
         branch_or_prnum=BRANCH_OR_PR_NUMBER,
-        branch=MAIN_BRANCH,
+        main_branch=MAIN_BRANCH,
```

Comment:
The argument `branch` was changed to `main_branch`, which is more descriptive. Just ensure all usages in the project are updated accordingly.